docs/develop/specify-office-hosts-and-api-requirements.md

```diff
@@ -93,16 +93,16 @@ The  **Hosts** element can contain one or more **Host** elements. The **Host** e
 
 ## Set the Requirements element in the manifest
 
-The  **Requirements** element specifies the minimum requirement sets or API members that must be supported by the Office host to run your add-in. The **Requirements** element can specify both requirement sets and individual methods used in your add-in. In version 1.1 of the add-in manifest schema, the **Requirements** element is optional for all add-ins, except for Outlook add-ins.
+The  **Requirements** element specifies the minimum requirement sets or API members that must be supported by the Office host to run your add-in. The **Requirements** element can specify both requirement sets and individual methods used in your add-in. In version "1.1" of the add-in manifest schema, the **Requirements** element is optional for all add-ins, except for Outlook add-ins.
```

Comment:
the quotes around "1.1" aren't needed here -- because it's referring to the version of the manifest schema (not to a numbered requirement set)